### PR TITLE
Add label.SetSocketCreate method

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@runcom @mrunalp PTAL

**Comment by rhatdan** [comment]
@giuseppe @vrothberg PTAL

**Comment by rhatdan** [comment]
@mheon PTAL

**Comment by vrothberg** [comment]
LGTM

File: go-selinux/selinux_stub.go [hunk 1]
```diff
@@ -96,6 +96,14 @@ func SetExecLabel(label string) error {
...
 }
 
+/*
+SetSocketLabel sets the SELinux label that the kernel will use for any programs
+that are executed by the current process thread, or an error.
+*/
+func SetSocketLabel(label string) error {
+	return nil
+}
+
```
Comment (edsantiago) [inline go-selinux/selinux_stub.go]: Please just ignore if this is a stupid question: Is there no need here for `SocketLabel()`?

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by giuseppe** [comment]
LGTM, but no privileges for merging

**Comment by mheon** [comment]
I share the question from @edsantiago - otherwise LGTM. As Giuseppe I have no merge privileges.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**